Kitodo/src/main/webapp/newpages/ProjekteBearbeiten.xhtml

```diff
@@ -79,7 +80,7 @@
                                                     infoClass="text_blue" showDetail="true" showSummary="true"
                                                     tooltip="true"/>
 
-                                        <!-- Box für die Bearbeitung der Details -->
+                                        <!-- Box f�r die Bearbeitung der Details -->
```

Comment:
The file `ProjekteBearbeiten.xhtml` is obviously used inside the application, isn't it? And although its comments should be ignored when that file is used, I know that this is not always true. There are parsers which get confused by unexpected characters in comments.